setup.py

```
@@ -2,7 +2,14 @@
 
 from setuptools import setup
 
-VERSION = '0.7.1'
+try:
+    with open('sporestack/__init__.py') as f:
+        for line in f:
+            if line.startswith('__version__'):
+                VERSION = line.replace("'", '').split('=')[1].strip()
+                break
+except:
+    print('Version not set.')
```

This try/except block is not great. The reason is that you want setup.py to fail hard. Printing an error and moving on is something I might not notice. Also, if __version__ happens to not exist in __init__.py, the exception won't even throw at all.

I would get rid of the try/except. After the break and with section, maybe do an `if VERSION is None`, `raise ValueError('__version not found in __init__.py')`.

Now even that isn't correct because it'll throw a different exception because the variable isn't set at all. So you can do something to detect if the variable exists at all and throw a variable based on that. Or, define VERSION as None or False right above.